tests/test_models.py

```diff
@@ -9,6 +9,7 @@
 import datetime
 import urlparse
 import itsdangerous
+import random, string
```

Comment:
Please break this into two lines.

ref: https://www.python.org/dev/peps/pep-0008/#imports